.gitignore

```diff
@@ -26,7 +26,17 @@ config.h.in
 config.h.in~
```

Comment:
Please update commit message. Please note autoconf 2.69 is mentioned as required - which is not the case